subsys/bluetooth/audio/has.c

```diff
@@ -514,12 +678,22 @@ int bt_has_preset_register(const struct bt_has_preset_register_param *param)
 		BT_WARN("param->name is too long (%zu > %u)", name_len, BT_HAS_PRESET_NAME_MAX);
 	}
 
+	CHECKIF(param->ops == NULL) {
+		BT_ERR("param->ops is NULL");
+		return -EINVAL;
+	}
+
+	CHECKIF(param->ops->select == NULL) {
+		BT_ERR("param->ops->select is NULL");
```

Comment:
I prefer using `BT_DBG` instead of `BT_ERR` when we are already returning an error code to the caller